src/components/RightSidebar/RightSidebar.vue

```diff
@@ -100,6 +100,37 @@
 				type="room"
 				:name="conversation.displayName" />
 		</AppSidebarTab>
+		<!-- Guest username setting form -->
+		<form
+			v-if="!getUserId"
+			class="username-form"
+			@submit.prevent="handleChooseUserName">
+			<h3>
+				{{ t('spreed', 'Display name: ') }} <strong>{{ this.actorDisplayName ? this.actorDisplayName : t('spreed', 'Guest') }}</strong>
```

Comment:
Trailing spaces dont work in translations, but you already have one after the translation